maps/guide-to-other.map

```diff
@@ -0,0 +1,4 @@
+~^/miscellaneous/how-to-clone-and-setup-the-free-code-camp-website-on-a-windows-pc/?$ https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally; #353
+~^/miscellaneous/guidelines-for-translating-free-code-camp-to-any-language/?$ https://contribute.freecodecamp.org/#/index?id=can-i-translate-freecodecamp39s-curriculum; #506
+~^/meta/free-code-camp-logo/?$ https://design-style-guide.freecodecamp.org/#logo; #657
+~^/miscellaneous/translate-free-code-camp-to-your-language/?$ https://contribute.freecodecamp.org/#/index?id=can-i-translate-freecodecamp39s-curriculum; #805
```

Comment:
```suggestion
~^/miscellaneous/how-to-clone-and-setup-the-free-code-camp-website-on-a-windows-pc/?$ https://contribute.freecodecamp.org; #353
~^/miscellaneous/guidelines-for-translating-free-code-camp-to-any-language/?$ https://contribute.freecodecamp.org; #506
~^/meta/free-code-camp-logo/?$ https://design-style-guide.freecodecamp.org/#logo; #657
~^/miscellaneous/translate-free-code-camp-to-your-language/?$ https://contribute.freecodecamp.org; #805
```

Let's drop the specificity, because it can break quiet easily. Also IMHO its not needed, since guide comes up in search results.

Reply:
Sure, that sounds good. The guides will probably be updated a lot more than any of the News ports.